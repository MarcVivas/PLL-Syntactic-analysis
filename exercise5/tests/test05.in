// Correct
S : Ac | fA | B
;
B : b 
;
A : i | a | B 
;
