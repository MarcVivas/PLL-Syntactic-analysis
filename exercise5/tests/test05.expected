A= a b i
B= b
S= a b f i

